### PR TITLE
Add a Jenkinsfile for the new dockerized Jenkins instance

### PR DESCRIPTION
**Pull request by mpol**

### CODE REVIEW

File: Jenkinsfile
Comment (marekhorst) [inline Jenkinsfile]: I think primer is quite "stable" in terms of code changes so we might drop cron trigger as well.

File: Jenkinsfile
Comment (marekhorst) [inline Jenkinsfile:30]: Is the `maven.test.failure.ignore` just a temporary flag? Or was it propagated from the current build definition?

**Review by marekhorst — APPROVED** [review]
Just made two comments: one question and one idea. Approving, but if you consider those comments as valid then you can introduce the changes before merging.

File: Jenkinsfile
Comment (mpol) [reply Jenkinsfile:30]: This is the way I found to distinguish between failing and unstable (tests failing) builds. The non-pipeline Jenkins Maven projects do this automatically somehow, here we let the maven step succeed and then the junit step will mark the build as unstable. We may not actually need this failing/unstable distinction, but Jenkins has it since forever.

Maybe https://plugins.jenkins.io/pipeline-maven/ would handle this automatically, but it is not installed and there are posts (eg. on stackoverflow) that suggests that people still need to use maven.test.failure.ignore with this plugin.

File: Jenkinsfile
Comment (mpol) [reply Jenkinsfile]: ok

**Pull request merged**
